service-worker/jitsi-enhancer.js

```diff
@@ -334,6 +376,28 @@ const createSuperText = (content, length = 5) => {
   }, length * 1000);
 };
 
+const createFlyingGif = (length = 8) => {
+  const videoWindow = document.querySelector("#jitsi-enhance-animation-container");
+
+    if (!videoWindow) return;
+
+    //csi.gif is included in /images if you want to implement this in a different way -Luna
```

Comment:
@constbeecoding We'd have to load it internally. 
Google Extensions forbid cdns and third-party calls like this.
https://developer.chrome.com/docs/apps/contentSecurityPolicy/#resourceLoading

I **might** be able to get away with it by declaring it in the manifest, but it'll add another permission & more eyeballs on this project from the Chrome Extension reviewers.

Will swap it over to internal files.

Reply:
Thank you! This was super fun, never explored how chrome extensions are set up! Well dang! I'm glad there is a way to implement this at least! Thanks @RockyKev !